src/VisualStudio/VisualBasic/Impl/Options/StyleViewModel.vb

```diff
@@ -361,5 +543,42 @@ End Class"
             Me.CodeStyleItems.Add(New BooleanCodeStyleOptionViewModel(CodeStyleOptions.PreferNullPropagation, ServicesVSResources.Prefer_null_propagation, s_preferNullPropagation, s_preferNullPropagation, Me, optionSet, nothingPreferencesGroupTitle))
             Me.CodeStyleItems.Add(New BooleanCodeStyleOptionViewModel(CodeStyleOptions.PreferIsNullCheckOverReferenceEqualityMethod, BasicVSResources.Prefer_Is_Nothing_over_ReferenceEquals, s_preferIsNothingCheckOverReferenceEquals, s_preferIsNothingCheckOverReferenceEquals, Me, optionSet, nothingPreferencesGroupTitle))
         End Sub
+
+        Private Sub AddParenthesesOptions(optionSet As OptionSet)
+            AddParenthesesOption(
+                LanguageNames.VisualBasic, optionSet, CodeStyleOptions.ArithmeticOperationParentheses,
+                BasicVSResources.Arithmetic_operations,
+                {s_arithmeticParenthesesIgnore, s_arithmeticParenthesesRequireForPrecedenceClarity, s_arithmeticParenthesesRemoveIfUnnecessary},
+                allowRequireForClarity:=True,
+                recommended:=True)
+
+            AddParenthesesOption(
+                LanguageNames.VisualBasic, optionSet, CodeStyleOptions.ShiftOperationParentheses,
+                BasicVSResources.Shift_operations,
+                 {s_shiftParenthesesIgnore, s_shiftParenthesesRequireForPrecedenceClarity, s_shiftParenthesesRemoveIfUnnecessary},
+                allowRequireForClarity:=True,
+                recommended:=False)
+
+            AddParenthesesOption(
+                LanguageNames.VisualBasic, optionSet, CodeStyleOptions.RelationalOperationParentheses,
+                BasicVSResources.Relational_operations,
+                {s_relationalParenthesesIgnore, s_relationalParenthesesRequireForPrecedenceClarity, s_relationalParenthesesRemoveIfUnnecessary},
+                allowRequireForClarity:=True,
+                recommended:=True)
+
+            AddParenthesesOption(
+                LanguageNames.VisualBasic, optionSet, CodeStyleOptions.LogicalOperationParentheses,
+                BasicVSResources.Logical_operations,
+                {s_logicalParenthesesIgnore, s_logicalParenthesesRequireForPrecedenceClarity, s_logicalParenthesesRemoveIfUnnecessary},
+                allowRequireForClarity:=True,
+                recommended:=True)
+
+            AddParenthesesOption(
+                LanguageNames.VisualBasic, optionSet, CodeStyleOptions.OtherOperationParentheses,
+                ServicesVSResources.Other_operations,
+                 {s_otherParenthesesIgnore, s_otherParenthesesRemoveIfUnnecessary},
```

Comment:
Super nit: the indent is different on this line, as well as 558 above.

Reply:
fixed.  #resolved.